app/jobs/populate_fake_data_job.rb

```diff
@@ -0,0 +1,160 @@
+require 'factory_bot_rails'
+require 'faker'
+require File.expand_path('spec/utils')
+
+class PopulateFakeDataJob
+  include Sidekiq::Worker
+  include Sidekiq::Throttled::Worker
+
+  sidekiq_options queue: :low
+  sidekiq_throttle threshold: { limit: 20, period: 1.minute }
+
+  attr_reader :user
+
+  FAKE_DATA_USER_ROLE = 'Seeded'.freeze
+  HOST = URI.parse("#{ENV['SIMPLE_SERVER_HOST_PROTOCOL']}://#{ENV['SIMPLE_SERVER_HOST']}").to_s
+  DEFAULT_HEADERS = { 'Content-Type' => 'application/json', 'ACCEPT' => 'application/json' }.freeze
+
+  TRAITS = {
+    newly_registered_patient: {
+      time_fn: -> { Faker::Time.between(1.month.ago, Time.now) },
+      size_fn: -> { rand(50..200) },
+      request_key: :patients
+    },
+    ongoing_bp: {
+      time_fn: -> { Faker::Time.between(1.month.ago, Time.now) },
+      size_fn: -> { rand(1..3) },
+      request_key: :blood_pressures,
+      patient_sample_size: 0.5
+    },
+    retroactive_bp: {
+      time_fn: -> { Faker::Time.between(12.months.ago, 1.month.ago.beginning_of_month) },
+      size_fn: -> { rand(2..5) },
+      request_key: :blood_pressures,
+      patient_sample_size: 0.5
+    },
+    scheduled_appointment: {
+      size_fn: -> { 1 },
+      request_key: :appointments,
+      patient_sample_size: 0.5
+    },
+    overdue_appointment: {
+      size_fn: -> { 1 },
+      request_key: :appointments,
+      patient_sample_size: 0.5
+    },
+    completed_phone_call: {
+      time_fn: -> { Faker::Time.between(6.months.ago, Date.today) },
+      size_fn: -> { rand(1..10) },
+      patient_sample_size: 0.5
+    }
+  }.freeze
+
+  def perform(user_id)
+    @user = User.find(user_id)
```

Comment:
`Rails.env.production?` won't work since all non-dev environments run in "production" mode. We've added a check against the env var `SIMPLE_SERVER_ENV` to be extra careful. As far as the job is concerned, it's stuck against the role `fake_data` which is only given to the sandbox boxes, so the cron job will never run this elsewhere.